test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -448,6 +466,20 @@
       (revoke-existing-cert! settings "localhost")
       (is (true? (revoked? cert))))))
 
+(deftest revoke-with-bundled-ca-certs
+  (testing "The CA certificate file can be a bundle when revoking a certificate"
+    (let [settings (testutils/ca-sandbox! cadir)
+          cert     (-> (:signeddir settings)
```

Comment:
and by "here" I mean this PR, I think we want to not align values in let blocks in general